app/code/community/FrontCommerce/Payline/Helper/Duplication.php

```diff
@@ -154,7 +334,7 @@ public function placeOrderFromQuoteAndPaylineToken(
           // Incorrect order_id
           throw new Exception('Quote getReservedOrderId (' . $quote->getReservedOrderId() . ') do not match tokenData (' . $orderIncrementId . ') for quoteId:' . $quote->getId());
         }
-      } elseif (!$this->getRequest()->getParam('paylineshortcut')) {
+      } else/* if (!$this->getRequest()->getParam('paylineshortcut')) */ {
         // Order should not be created exept from shortcut
```

Comment:
1. commenting code is a very bad practice
1. the inline comment seems to be irrelevant now